### PR TITLE
Update hardware-matrix.md for VP4670 ver1

### PR DESCRIPTION
**Pull request by Psotas**

### CODE REVIEW

File: docs/variants/protectli_vp46xx/hardware-matrix.md [hunk 1]
```diff
@@ -54,7 +54,31 @@ coreboot port on the Protectli VP46XX firewall.
...
+| **Wireless card**              | JJPLUS JWW6051                              |
+| **4G Modem**                   | AMIT MDG200-0T001                           |
+| **Display**                    | Display 1: HDMI 1920x1080p                  |
+|                                | Display 2: DP 1920x1080p                    |
+| **Ethernet controller**        | 6x intel i225 (on-board)                    |
+| **Internal devices**           | Audio subsystem                             |
+| **TPM**                        | Protectli TPM                               |
+| **Network**                    | Local network wired connection              |
+| **Remote Testing Environment** | RTE v1.1.0                                  |
+| **Power Control**              | RTE v1.1.0                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp46xx/hardware-matrix.md:78]: As power control should be set Sonoff S26. The same in the `Protectli VP4670 ver2` config

File: docs/variants/protectli_vp46xx/hardware-matrix.md [hunk 2]
```diff
@@ -54,7 +54,31 @@ coreboot port on the Protectli VP46XX firewall.
 | **Power Control**              | RTE v1.1.0                                  |
 | **Power supply**               | Channel Well Technology 12V, 7.5A 90W       |
 
-## Protectli VP4670
+## Protectli VP4670 ver1
+
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel Core i7-10810U                        |
+| **RAM**                        | Slot 1: Empty                               |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp46xx/hardware-matrix.md:62]: @Psotas could we fill this slot? Platforms without enough RAM might work very very slowly.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_vp46xx/hardware-matrix.md (hunk 1 above)
Comment (Psotas) [reply docs/variants/protectli_vp46xx/hardware-matrix.md]: Changed to:
| **Power Control**              | Sonoff S26                                  |

File: docs/variants/protectli_vp46xx/hardware-matrix.md (hunk 2 above)
Comment (Psotas) [reply docs/variants/protectli_vp46xx/hardware-matrix.md]: Ram swaped to:
| **RAM**                        | Slot 1: CRUCIAL CT16G4SFRA32A               |
|                                | Slot 2: CRUCIAL CT16G4SFRA32A               |

File: docs/variants/protectli_vp46xx/hardware-matrix.md [hunk 3]
```diff
@@ -54,7 +54,31 @@ coreboot port on the Protectli VP46XX firewall.
...
+| **Wireless card**              | JJPLUS JWW6051                              |
+| **4G Modem**                   | AMIT MDG200-0T001                           |
+| **Display**                    | Display 1: HDMI 1920x1080p                  |
+|                                | Display 2: DP 1920x1080p                    |
+| **Ethernet controller**        | 6x intel i225 (on-board)                    |
+| **Internal devices**           | Audio subsystem                             |
+| **TPM**                        | Protectli TPM                               |
+| **Network**                    | Local network wired connection              |
+| **Remote Testing Environment** | RTE v1.1.0                                  |
+| **Power Control**              | Sonoff S26                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp46xx/hardware-matrix.md:78]:
```suggestion
| **Power control**              | Sonoff S26                                  |
```

File: docs/variants/protectli_vp46xx/hardware-matrix.md [hunk 4]
```diff
@@ -75,5 +99,5 @@ coreboot port on the Protectli VP46XX firewall.
 | **TPM**                        | Protectli TPM                               |
 | **Network**                    | Local network wired connection              |
 | **Remote Testing Environment** | RTE v1.1.0                                  |
-| **Power Control**              | RTE v1.1.0                                  |
+| **Power Control**              | Sonoff S26                                  |
```
Comment (sulewskiprzemyslaw) [inline docs/variants/protectli_vp46xx/hardware-matrix.md:102]:
```suggestion
| **Power control**              | Sonoff S26                                  |
```

**Review by macpijan — APPROVED** [review]

**Pull request merged**